int32[] agentNeighbors
float32[] agentNeighborsDistance
VoInfo[] agentNeighborsVoInfo
int32[] obstacleNeighbors
float32[] obstacleNeighborsDistance
VoInfo[] obstacleNeighborsVoInfo
geometry_msgs/Polygon voPoly
bool[] inCollisionWith